lemur/plugins/lemur_verisign/plugin.py

```diff
@@ -221,8 +221,12 @@ def create_certificate(self, csr, issuer_options):
                 extra={"common_name": issuer_options.get("common_name", "")}
             )
             raise Exception(f"Error with Verisign: {response.content}")
-        # TODO add external id
-        return cert, current_app.config.get("VERISIGN_INTERMEDIATE"), None
+        # DONE: TODO add external id
```

Comment:
we can remove this comment

Reply:
Hi @hosseinsh :good suggestions. Should I adapt the documentation as well?

Reply:
that would be excellent, thank you!